pages/1.13/release-notes/1.13.1/index.md

```diff
@@ -0,0 +1,103 @@
+
+### Admin Router
+- Changes the maximum size allowed for uploads to a service through Admin Router (COPS-4651, DCOS-52768).
+
+    This release increases the maximum size allowed for uploading packages from 1GB to 16GB. This change enables you to upload larger packages to a registry service without timing out the upload connection.
+
+<!--### Command-line interface (CLI)
+- 
+
+### Diagnostics and logging
+- 
+
+### GUI --> 
+
+### Installation
+- Fixes issues that caused some DC/OS components to crash when the `/tmp` directory is mounted using the `noexec` option (DCOS-53077).
+
+<!--### Job management and scheduling --> 
+
+### Marathon
+- Improves handling for tasks that return a TASK_UNKNOWN state (COPS-4883). 
+
+    In most cases, the TASK_UNKNOWN state results when there are explicit reconciliation requests for:
+    - Unrecognized tasks on registered agents
+    - Tasks requests on unregistered nor unreachable agents
+
+    Prior to this fix, any app instance that returned `TASK_UNKNOWN` as a Mesos task state could cause the DC/OS API to fail with the following error:
+
+    “TASK_UNKNOWN is an unknown Mesos task state”
+    
+    With this release, the TASK_UNKNOWN state is recognized by other DC/OS components. This state no longer causes operational failures or error messages that require the task to be deleted and relaunched as a new task with a new unique ID.
+
+- Fixes an issue with scaling Marathon apps when using a persistent volume (COPS-4892).
+
+    In some cases, Marathon tasks that used the Docker containerizer and persistent volumes could not be scaled back up to one instance after the app was suspended by scaling the instance count to zero. Before this fix, the task could be restarted but would lose its persistent volume. 
+    
+    The fix in this release ensure that the Marathon app can be suspended then scaled back up to a running instance without losing the persistent volume.
+
+### Metrics
+- Enables framework names to be properly decoded in metric tags (DCOS_OSS-5039).
+
+    Mesos masters allow spaces in framework names by using percent-encoding (%20) of the framework name. This release updates the Telegraf plugin to enable it to decode the framework name and export metrics with the correct tags.
+
+### Networking
+- Adds round-robin DNS support so that DNS requests do not always return address (A) records in the same order (DCOS_OSS-5118).
+
+- Returns canonical name (CNAME) records before address (A or AAAA) records in DNS responses (DCOS_OSS-5108).
+
+    For most DNS clients, the order in which records are returned has no affect. However, there are some DNS clients that require CNAME records to be listed before A records. This change resolves issues for DNS clients that have this requirement.
+
+- Fixes a problem with the `dcos-iam-ldap-sync` service failing to start correctly after a system reboot (COPS-4455, COPS-4814, DCOS-48107, DCOS-53420).
```

Comment:
I think this one should be under the ### Security section, it's not a networking related issue.

Reply:
good call. fixed.